ocs_ci/framework/conf/default_config.yaml

```diff
@@ -41,7 +41,7 @@ DEPLOYMENT:
   # opened issue here about the version in the name of OLM
   # https://github.com/openshift/ocs-operator/issues/50
   ocs_csv_version: "v0.0.1"
-  ocs_operator_storage_cluster_cr: "https://raw.githubusercontent.com/openshift/ocs-operator/master/deploy/crds/ocs_v1alpha1_storagecluster_cr.yaml"
+  ocs_operator_storage_cluster_cr: "https://raw.githubusercontent.com/openshift/ocs-operator/master/deploy/crds/ocs_v1_storagecluster_cr.yaml"
```

Comment:
this also returns 404

Reply:
@RazTamir do you mean the updated version? For me the updated link is working: https://raw.githubusercontent.com/openshift/ocs-operator/master/deploy/crds/ocs_v1_storagecluster_cr.yaml

Reply:
strange, now it is working